core-services/release-controller/README.md

```diff
@@ -53,10 +53,14 @@ To abort/remove a release tag:
 
 * `deploy-*.yaml` - the controller definitions
 * `images-*.yaml` - definitions of the image streams used by the publish / build rules
-* `releases/release-*.json` - definitions of the build and publishing rules for sets of images
-* `repos/ocp-*-*.repo` - definitions where RPMs are loaded from for OCP builds
+* `_releases/release-*.json` - definitions of the build and publishing rules for sets of images
+* `_repos/ocp-*-*.repo` - definitions where RPMs are loaded from for OCP builds
 * `rpms-*.yaml` - definitions of the services that exposed mirrored RPMs within the clutser
 
+### Release informers
```

Comment:
```suggestion
### TestGrid Dashboard Updates
```

Reply:
This section is about release-controller jobs, and mentions TestGrid dashboard updates in passing.  Should I use something more generic that includes both blocking and informing jobs?  Maybe `Release jobs`?

Reply:
this section (`###`) covers nothing but testgrid behavior/config.

if you were trying to document how the informing+blocking jobs are configured for the release-controller, then 
1) let's call it `Release Gating Jobs`
2) make sure you cover how you make the job an informing job vs a blocking job (the `optional` flag i believe)
3) include a subsection (`####`) that tells people to go update testgrid whenever they mess w/ the release job list.  or maybe just a separate paragraph at the end of the (`###`) section.

Reply:
> let's call it Release Gating Jobs

Done with 1e14c589f -> 67f3c37b8, although I went with sentence-case `Release gating jobs`.  Master is not consistent for h3, but slants sentence-case:

```console
$ git --no-pager grep -h '^#' origin/master -- core-services/release-controller/README.md 
### Artifacts created:
### Initial configuration
### Admin actions:
### Files in this folder
### RPM Mirrors
````

> make sure you cover how you make the job an informing job vs a blocking job (the `optional` flag i believe)

Done in 1e14c589f -> 67f3c37b8.

> ... or maybe just a separate paragraph...

Split out into a separate paragraph with 1e14c589f -> 67f3c37b8.